src/redux/configureStore.js

```diff
@@ -1,12 +1,18 @@
-import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import { configureStore } from '@reduxjs/toolkit';
 import bookreducer from './books/books';
 import categoriesReducer from './categories/categories';
 
-const root = combineReducers({
-  bookreducer,
-  categoriesReducer,
+// const root = combineReducers({
+//   bookreducer,
+//   categoriesReducer,
+const store = configureStore({
+  reducer: {
+    books: bookreducer,
+    categories: categoriesReducer,
+  },
+
 });
 
-const store = configureStore({ reducer: root });
+// const store = configureStore({ reducer: root });
```

Comment:
- [ ] As I mentioned in the above comments, kindly make sure you remove the commented and unused code lines from your codebase to make it more professional and for **[best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)**.